lib/network/network_request.dart

```diff
@@ -0,0 +1,13 @@
+enum HttpMethod { get, post }
```

Comment:
don't use `get` as it's registered keyword(

As example you can use `getMethod` and `postMethod` and add associative value like that. I think it should work, instead of convert name to uppercase.

```suggestion
enum HttpMethod {
getMethod('GET), 
postMethod('POST'),

final String value;

const HttpMethod(this.value);

}
```

I didn't check this in VS Code, you can give it a try